plugins/Jobs.h

```diff
@@ -15,4 +15,8 @@ class BedrockPlugin_Jobs : public BedrockPlugin {
     string _constructNextRunDATETIME(const string& lastScheduled, const string& lastRun, const string& repeat);
     bool _validateRepeat(const string& repeat) { return !_constructNextRunDATETIME("", "", repeat).empty(); }
     bool _hasPendingChildJobs(SQLite& db, int64_t jobID);
+
+    // Validates the sqlite date modifiers
+    // See: https://www.sqlite.org/lang_datefunc.html
```

Comment:
Why put this comment here in the `.h` file when all the other functions have no descriptions?  I suggest keeping this in the first line of the function in the `.cpp` file, so someone reading through the function knows about this link.